ethcore/src/engines/clique/mod.rs

```diff
@@ -0,0 +1,643 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
+
+// Parity Ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Implementation of Clique (POA) Engine.
+//!
+//! mod.rs -> CliqueEngine, the engine api implementation, with additional block state tracking.
+//!
+//! CliqueEngine -> {
```

Comment:
I can't find a `CliqueEngine` struct in this file, is this supposed to be `Clique` instead?

Reply:
Fixed.